aten/src/ATen/native/cudnn/BatchNorm.cpp

```diff
@@ -98,6 +103,21 @@ cudnnBatchNormMode_t getCudnnBatchNormMode(
 
 } // namespace
 
+size_t _get_cudnn_batch_norm_reserve_space_size(const Tensor& input_t) {
+  size_t reserve_size;
+  TensorArg input{input_t, "input", 1};
+  TensorDescriptor idesc{*input, 4};
+  auto handle = getCudnnHandle();
+  cudnnBatchNormMode_t mode = getCudnnBatchNormMode(
+      true, // training
```

Comment:
Why is this always true?

Reply:
Hmm yeah maybe we should expose this as an arg. This was copied from the existing cudnn code and only used in the training case, but I guess our no_update decomp now also calls this